tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "moduleResolution": "Node",
     "isolatedModules": false,
     "jsx": "react",
+    "strict": true,
```

Comment:
did it work for you with `"strict": true`, I get compilation errors in container components.